index.js

```diff
@@ -1,5 +1,8 @@
 const express = require('express');
 const path = require('path');
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
I don't think importing dotenv is necessary here again, this is a repetition if you've already done in the webpack config. or does it not work without this?

Reply:
I was following 

> <img alt="screen shot 2018-12-16 at 4 00 20 pm" width="1440" src="https://user-images.githubusercontent.com/31810086/50055240-5e22ec00-014c-11e9-8e32-84a9d1de3f09.png">
> 
> Test to make sure this feature actually works, I clicked the button and it got stuck with this error and now the whole app is stuck with that over-laid spinner.
> Update: after manually clearing local storage and refreshing it works fine, this may be helpful in finding the bug.

This error is only thrown when there is no GOOGLE_CLIENT_ID or when you switch branches without restarting your server, then the App won't be able to pick your env variables. I highly doubt this had anything to do with local storage.

Reply:
I was following @sulenchy. Since he had fixed the environment variable problems, I took the changes from his file.